examples/textConnect/config_js/config.js

```diff
@@ -3,42 +3,61 @@
  * Copyright (c) 2016 Cybozu
  *
  * Licensed under the MIT License
- */
+ // */
 jQuery.noConflict();
 (function($, PLUGIN_ID) {
     'use strict';
 
-    // プラグインIDの設定
+    //To switch the language used for instructions based on the user's launguage setting
+    var terms = {
+        'en': {
+            connectTitle: 'Fields to Connect',
+            connectDescription: 'Please select the fields to connect. (5 fields max)',
+            delimiterTitle: 'Delimiter',
+            delimiterDescription: 'Please specify the delimiter used between connected values. If not specified, values will be connected without delimiters.',
+            errorMessage: '"Fields to display the connected result" is a required setting. Please select the output fields where the connected results will be inserted.',
+            resultTitle: 'Fields to display the connected result',
+            resultDescription: 'Please select the fields to display the connected result.',
+            saveButton: 'Save',
+            cancelButton: 'Cancel'
+        },
+        'ja': {
+            connectTitle: '結合する項目',
+            connectDescription: '結合する項目を選択してください。（最大5つまで）',
+            delimiterTitle: '項目間の記号',
+            delimiterDescription: '結合される項目の間に表示される記号を入力してください。未選択の場合、各項目が直接結合されます。',
+            errorMessage: '「結合された文字列を表示する項目」は必須です。',
+            resultTitle: '結合された文字列を表示する項目',
+            resultDescription: '結合された文字列を表示する項目を選択してください。',
+            saveButton: '保存する',
+            cancelButton: 'キャンセル'
+        }
+    }
+    var lang = kintone.getLoginUser().language;
+    var i18n = (lang in terms) ? terms[lang] : terms['en'];
+
+    // set the pluginID
     var CONF = kintone.plugin.app.getConfig(PLUGIN_ID);
 
     function decodeSpace(htmlstr) {
         return htmlstr.replace(/&nbsp;/g, ' ').replace(/&emsp;/g, '　');
     }
 
     function setDefault() {
-        if (CONF) {
+        if (Object.keys(CONF).length > 0) {
             for (var i = 1; i < 16; i++) {
                 $('#select' + i).val(CONF['select' + i]);
             }
 
-            // 旧プラグインから設定を引き継ぐ
-            if (CONF.hasOwnProperty('line_number')) {
-                $('#copyfield1').val(CONF.copyfield);
-                if (CONF.copyfield !== '') {
-                    $('#between1').val(decodeSpace(CONF.between));
+            // get the previous plugin setting
```

Comment:
text_connect v1.1.1 (two versions before) contained a config property 'line_number' which is unused for further  versions. There is though, the possibility of text_connect v1.1.1 users upgrading their plug-in to the newest version. If we do not place in logic that will address this situation, and the user upgrades their plug-in, the previous settings that they set in the plug-in will be reset.
The new code written here has removed that logic.

Reply:
Reference:
https://github.com/kintone/plugin-examples/blob/b4b3dc28c502f3d398397d2958b50e1d0af1dd9f/examples/textConnect/config_js/config.js#L18-L43

Reply:
I reverted the changes made.